app/models/UpdateGoodsRecord.scala

```diff
@@ -67,11 +71,12 @@ object UpdateGoodsRecord {
   def validateCommodityCode(
     answers: UserAnswers,
     recordId: String,
-    isCategorised: Boolean
+    isCategorised: Boolean,
+    navigatingFromExpiredCommCode: Boolean
```

Comment:
I'd rename it to be "isComCodeExpired" or something rather than "navigatingFromExpired". Slight difference but it makes clear why we are doing this.